src/ontology/cl-edit.owl

```diff
@@ -28453,6 +28453,7 @@ EquivalentClasses(obo:CL_2000044 ObjectIntersectionOf(obo:CL_2000008 ObjectSomeV
 # Class: obo:CL_2000045 (foreskin melanocyte)
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:TermGenie") obo:IAO_0000115 obo:CL_2000045 "Any melanocyte of skin that is part of a skin of prepuce of penis.")
+AnnotationAssertion(obo:IAO_0000233 obo:CL_2000045 "https://github.com/obophenotype/cell-ontology/issues/1562")
```

Comment:
Tjank you for adding. This is still a literal, can you add as IRI? It is the third tab.

Reply:
Done